swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/TunnelStore.swift

```diff
@@ -12,11 +12,14 @@ import OSLog
 enum TunnelStoreError: Error {
   case tunnelCouldNotBeStarted
   case tunnelCouldNotBeStopped
+  case cannotSaveToTunnelWhenConnected
+  case cannotSignOutWhenConnected
+  case stopAlreadyBeingAttempted
 }
 
 public struct TunnelProviderKeys {
   static let keyAuthBaseURLString = "authBaseURLString"
-  static let keyAccountId = "accountId"
+  static let keyAccountId = "accountId"  // Unused
```

Comment:
Can this just be removed if we're not using it?
```suggestion
```

Reply:
We use it to remove the data from the tunnel configuration for tunnel configurations created by older versions of the app. I will clarify the comment.

Reply:
Fixed in commit e2e79d7.